da/da.go

```diff
@@ -25,6 +25,8 @@ type DAResult struct {
 	Code StatusCode
 	// Message may contain DA layer specific information (like DA block height/hash, detailed error message, etc)
 	Message string
+	// DataLayerHeight informs about a height on Data Availability Layer for given result.
+	DataLayerHeight uint64
```

Comment:
Weird that this is called `DataLayerHeight` instead of `DAHeight`

Reply:
Good point. Fixed, and also created #345 and https://github.com/celestiaorg/dalc/issues/79.